src/explore-education-statistics-common/src/components/InfoIconButton.tsx

```diff
@@ -0,0 +1,24 @@
+import ButtonText from '@common/components/ButtonText';
+import React, { MouseEventHandler } from 'react';
+import styles from './InfoIconButton.module.scss';
+
+interface Props {
+  description: string;
+  testId?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+const InfoIconButton = ({ description, testId, onClick }: Props) => {
+  return (
+    <ButtonText
+      className={styles.infoIconButton}
+      testId={testId}
+      onClick={onClick}
+    >
+      <span aria-hidden>?</span>
+      <span className="govuk-visually-hidden">{description}</span>
+    </ButtonText>
+  );
+};
```

Comment:
As mentioned earlier, we should refactor this component a bit:

- `InfoIconButton` should be renamed to `InfoIcon` and should literally just be an icon (no button).
- We can remove all the other props - only need `description`.